bridge-stp.in

```diff
@@ -212,7 +212,7 @@ case "$action" in
             # Add bridge to mstpd and configure.
             echo "Adding/configuring bridge '$bridge' ..."
             "$mstpctl" addbridge "$bridge" || continue
-            if [ -n "$config_cmd" ]; then
+            if [ -x "$config_cmd" ]; then
```

Comment:
I meant to use 'type -t ...' or command or something  maybe in combination with '[ -x ... ], not '[ -t ...]' to check for existence and executability. But I see this is not so easy to do in a portable way.

Yes I'm using it on OpenWrt. But not the original mstp_config_bridge script since it doesn't work anyway (no ifquery resp. python and /etc/network/interfaces). I wrote a simple config script which only forces RSTP on the bridge.

I have no clue however to get mstpd working with hardware switches. kernel/mstpd would have to know the switch topology and control each single port.... which looks kind of impossible to do with swconfig based switches.